README.md

```diff
@@ -26,8 +26,10 @@ This repository is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) that hol
 
 See [rush.json](./rush.json) for the complete list of packages and [Versioning.md](./Versioning.md) for package and API versioning policies.
 
-Each package has its own **node_modules** directory that contains symbolic links to *common* dependencies managed by Rush.
+## Quick Start
 
+- This is a [sample](https://www.itwinjs.org/sandboxes/iTwinPlatform/3d%20Viewer) of an iTwin viewer - a frontend application that displays infrastructure projects on browsers. It uses many of the APIs and libraries published from this repository.
```

Comment:
thoughts on hiding the code, and just showing the viewer?

https://www.itwinjs.org/sandbox/iTwinPlatform/3d%20Viewer?mainHeader=hidden&editorPane=hidden&headers=hidden

go to the ellipses > embed sandbox > mess around with the option & copy the iframe url

Reply:
our users will still be developers - it'll be nice to show them the coding panel - users can play around with the code, and the panel encourages their own discovery. (If the code is outdated, that's a different issue to tackle)